### PR TITLE
node-fix-error-in-test-suite-when-updating

### PR DESCRIPTION
**Pull request by chadiegilnerubia**
Ticket ID:

### CODE REVIEW

File: src/controllers/test-suite/test-item-controller.ts [hunk 1]
```diff
@@ -110,7 +110,7 @@ export const update = async (req: Request, res: Response) => {
     const user = req.user
 
     const { id } = req.params
-    const { apiId, http_method, payload, response, description } = req.body
+    const { apiId, status, http_method, payload, response, description } = req.body
```
Comment (jatnerubia) [inline src/controllers/test-suite/test-item-controller.ts:113]: move **status** after the **description**

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a minor change ^^

File: src/controllers/test-suite/test-item-controller.ts (hunk 1 above)
Comment (chadiegilnerubia) [reply src/controllers/test-suite/test-item-controller.ts]: done :3

**Review by jatnerubia — APPROVED** [review]
Cool

**Pull request merged**